### ISSUE TITLE
failure resolving implicit package name

### ISSUE THREAD

**Issue by FND** [faucet-pipeline/faucet-pipeline-static#23]
reduced test case:
```javascript
// faucet.config.js
module.exports = {
    static: [{
        source: "material-design-icons/iconfont",
        target: "./dist/fonts"
    }]
};
```
```shell
$ npm install -D faucet-pipeline-static
$ npm install material-design-icons
$ ./node_modules/.bin/faucet
ERROR: could not resolve `"material-design-icons/iconfont"`
```

yet `source: "./node_modules/material-design-icons/iconfont"` works as expected

it appears the error occurs in `require.resolve`: [faucet-core:resolve.js:38](https://github.com/faucet-pipeline/faucet-pipeline-core/blob/a758e25d283930e5a09a587797f5a85364421bd5/lib/util/resolve.js#L38) (via [l. 23](https://github.com/faucet-pipeline/faucet-pipeline-core/blob/a758e25d283930e5a09a587797f5a85364421bd5/lib/util/resolve.js#L23))
```
require.resolve("material-design-icons", { paths: ["/path/to/root"] })
```

this is because the `iconfont` sub-directory does not contain an `index.js` file - meaning faucet-core's `resolvePath` is insufficiently generic

tested with Node v11.9.0

**Comment by moonglum** [comment]
Does this still apply now that we use the real resolve due to dropping older Node versions? 🤔

**Comment by FND** [comment]
Well, given that the issue occurs with Node's own `require.resolve` (which expects an `index.js`, apparently) and was confirmed with v11, I would think it's still relevant.

I was uncertain, so have set up a minimal test case that confirms this:
https://github.com/faucet-pipeline/faucet-pipeline-core/commits/testcase-resolve
(note the commit messages and two different CI results)

**Comment by FND** [comment]
fixed in faucet-core v1.5.1 (https://github.com/faucet-pipeline/faucet-pipeline-core/commit/685eefa8c879f71e1ac5383c56d6db4b2fb75d11)
